actions/concurrent_runner.go

```diff
@@ -71,7 +70,7 @@ func NewConcurrentRunner(ctx context.Context, timeout time.Duration) *Concurrent
 }
 
 // Start starts an Action in a separate goroutine.
-func (r *ConcurrentRunner) Start(a Action) {
+func (r *ConcurrentRunner) Start(a Action, timeout *duration.Duration) {
```

Comment:
I wonder if `time.Duration` will fit better there. Not sure

Reply:
Actually, I think time.Duration is definitely better there – this package does not import protobuf otherwise

Reply:
Actually, I think time.Duration is definitely better there – this package does not import protobuf otherwise

Reply:
will change it

Reply:
Agree, I don't see reasons not to use `time.Duration` here

Reply:
+1